docs/troubleshooting/management/ts-management.asciidoc

```diff
@@ -0,0 +1,33 @@
+[[ts-management]]
+== Management tools
+
+This topic covers common troubleshooting issues when using the {elastic-sec} <<sec-manage-intro, management tools>>.
+
+[discrete]
+[[ts-endpoints]]
+=== Endpoints
+
+[discrete]
+[[ts-transform-failed]]
+.Required transform failed
+[%collapsible]
+====
+If you encounter a `“Required transform failed”` notice on the Endpoints page, you can usually resolve the issue by restarting the transform. See {ref}/transforms.html[Transforming data] for more information on transforms.
```

Comment:
Don't think you need the double quotes around the notice unless the message displays with quotes in the UI.

Reply:
I noticed similar messages formatted this way in the [other troubleshooting page](https://www.elastic.co/guide/en/security/7.15/ts-detection-rules.html), so I went with it. @jmikell821 WDYT?

Reply:
Since this is a specific error message, I'd leave the quotes in. If anyone sees different requirements in the style guide or elsewhere, let me know!